### ISSUE TITLE
Unable to change root password for database if provided via environment variable ORIENTDB_ROOT_PASSWORD

### ISSUE THREAD

**Issue by akshay-mahakalkar** [orientechnologies/orientdb#10273]
## Expected behavior  
If the value of ORIENTDB_ROOT_PASSWORD is changed in the new password should be consumed.

## Actual behavior  
When ORIENTDB_ROOT_PASSWORD is changed, database is unable to use new password, instead continues to use old password.
When the OSystem database is deleted then only new password is consumed.

## Steps to reproduce  
1. Download and start OrientDB server by running **ORIENTDB_ROOT_PASSWORD="password" server.sh** command.
2. Try accessing the studio. Should be accessible.
3. Change the password and run again. Should not be accessible.

## Extra Findings
If password is provided via orientdb-server-config.xml file as provided below
`    <users>
        <user name="root" password="Root@1234" resources="*" />
        <user name="guest" password="admin" resources="connect,server.listDatabases,server.dblist" />
    </users>`
Then I am able to login database with both passwords. One provided via environment variable and one provided in xml file at the same time.

**Comment by tglman** [comment]
Hi,
Yes I think that password as today is retrieve and stored somewhere else, I do agree this may not be the best behavior, I will see if this can be corrected.

**Comment by tglman** [comment]
Hi,

This should be fixed in 3.2.33 let me know if you can verify it.

Regards

**Comment by akshay-mahakalkar** [comment]
Hi @tglman,

Thanks for resolving the issue. I verified it is now resolved. If new password set via environment variable it is considering the new password.
Only one thing I found new is, now the database console logs shows the database password.
![image](https://github.com/user-attachments/assets/59ddcfb9-4978-4c9f-8620-a9ba222f3e15)
Was this intentional as previously it was "INFO  Found ORIENTDB_ROOT_PASSWORD variable, using this value as root's password [OServer]".
